### PR TITLE
Fixes multiline markdown link text

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Joins multiline split markdown link text.

### CODE REVIEW

File: files/en-us/web/api/range/comparenode/index.md [hunk 1]
```diff
@@ -30,8 +30,7 @@ The possible values are:
   - : Node starts after and ends before the Range, i.e. the Node is completely selected by
     the Range.
 
-> **Warning:** This method [has been
-> removed](/en-US/docs/Mozilla/Firefox/Releases/3/Site_compatibility) from [Gecko 1.9](/en-US/docs/Mozilla/Firefox/Releases/3) and
+> **Warning:** This method [has been > removed](/en-US/docs/Mozilla/Firefox/Releases/3/Site_compatibility) from [Gecko 1.9](/en-US/docs/Mozilla/Firefox/Releases/3) and
```
Comment (teoli2003) [inline files/en-us/web/api/range/comparenode/index.md:33]:
```suggestion
> **Warning:** This method [has been removed](/en-US/docs/Mozilla/Firefox/Releases/3/Site_compatibility) from [Gecko 1.9](/en-US/docs/Mozilla/Firefox/Releases/3) and
```

File: files/en-us/web/api/window/getcomputedstyle/index.md [hunk 2]
```diff
@@ -50,10 +50,8 @@ object, which updates automatically when the element's styles are changed.
 
     > **Note:** Valid pseudo-element selector refers to syntactic
     > validity, e.g. `::unsupported` is considered valid, even though the
-    > pseudo-element itself is not supported.  Additionally, the latest W3 standard [explicitly
-    > supports](https://www.w3.org/TR/cssom-1/#dom-window-getcomputedstyle) only `::before` and `::after`, while the CSS
-    > WG draft [does not
-    > restrict this value](https://drafts.csswg.org/cssom/#dom-window-getcomputedstyle).  Browser compatibility may vary.
+    > pseudo-element itself is not supported.  Additionally, the latest W3 standard [explicitly > supports](https://www.w3.org/TR/cssom-1/#dom-window-getcomputedstyle) only `::before` and `::after`, while the CSS
```
Comment (teoli2003) [inline files/en-us/web/api/window/getcomputedstyle/index.md:53]:
```suggestion
    > pseudo-element itself is not supported.  Additionally, the latest W3 standard [explicitly supports](https://www.w3.org/TR/cssom-1/#dom-window-getcomputedstyle) only `::before` and `::after`, while the CSS
```

File: files/en-us/web/api/window/open/index.md [hunk 3]
```diff
@@ -399,8 +393,7 @@ disorienting to users (Back toolbar button is disabled).
 > a small monitor where the windows are maximized to fill up the screen. So a user who
 > tries to return to the origin will be confused by a grayed out _Back_
 > button."
-> quote from [The Top Ten
-> _New_ Mistakes of Web Design](https://www.useit.com/alertbox/990530.html): 2. Opening New Browser Windows, Jakob
+> quote from [The Top Ten >_New_Mistakes of Web Design](https://www.useit.com/alertbox/990530.html): 2. Opening New Browser Windows, Jakob
```
Comment (teoli2003) [inline files/en-us/web/api/window/open/index.md:396]:
```suggestion
> quote from [The Top Ten New Mistakes of Web Design](https://www.useit.com/alertbox/990530.html): 2. Opening New Browser Windows, Jakob
```

File: files/en-us/web/api/window/requestanimationframe/index.md [hunk 4]
```diff
@@ -127,12 +127,8 @@ Edge versions below 17 and Internet Explorer do not reliably fire
 - {{domxref("Window.cancelAnimationFrame()")}}
 - [mozRequestAnimationFrame](http://weblogs.mozillazine.org/roc/archives/2010/08/mozrequestanima.html)
   \- Blog post
```
Comment (teoli2003) [inline files/en-us/web/api/window/requestanimationframe/index.md:129]:
```suggestion
- [mozRequestAnimationFrame](http://weblogs.mozillazine.org/roc/archives/2010/08/mozrequestanima.html) –  Blog post
```

File: files/en-us/web/api/window/sessionstorage/index.md [hunk 5]
```diff
@@ -100,8 +99,7 @@ field.addEventListener("change", function() {
 });
 ```
 
-> **Note:** Please refer to the [Using the Web
-> Storage API](/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API) article for a full example.
+> **Note:** Please refer to the [Using the Web > Storage API](/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API) article for a full example.
```
Comment (teoli2003) [inline files/en-us/web/api/window/sessionstorage/index.md:102]:
```suggestion
> **Note:** Please refer to the [Using the Web Storage API](/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API) article for a full example.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Great! Your script don't deal correctly when the multiline link is inside a quote or a note, but there are few of such cases, so that's easy to correct.

**Comment by OnkarRuikar** [comment]
The pattern just replaced `\n` in the text. 😋

I'll keep updating it as we find more variations. As the changes are so simple they got overlooked easily.

File: files/en-us/web/api/window/requestanimationframe/index.md [hunk 6]
```diff
@@ -125,8 +125,7 @@ Edge versions below 17 and Internet Explorer do not reliably fire
 ## See also
 
 - {{domxref("Window.cancelAnimationFrame()")}}
-- [mozRequestAnimationFrame](http://weblogs.mozillazine.org/roc/archives/2010/08/mozrequestanima.html)
-  \- Blog post
+- [mozRequestAnimationFrame](http://weblogs.mozillazine.org/roc/archives/2010/08/mozrequestanima.html) –  Blog post
```
Comment (teoli2003) [inline files/en-us/web/api/window/requestanimationframe/index.md:128]:
```suggestion
- [mozRequestAnimationFrame](http://weblogs.mozillazine.org/roc/archives/2010/08/mozrequestanima.html) – Blog post
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**